echopype/convert/parse_base.py

```diff
@@ -159,11 +146,8 @@ def parse_raw(self):
             # Read the rest of datagrams
             self._read_datagrams(fid)
 
-        if "ALL" in self.data_type:
-            # Convert ping time to 1D numpy array, stored in dict indexed by channel,
-            #  this will help merge data from all channels into a cube
-            for ch, val in self.ping_time.items():
-                self.ping_time[ch] = np.array(val, dtype="datetime64[ns]")
+        for ch, val in self.ping_time.items():
+            self.ping_time[ch] = np.array(val, dtype="datetime64[ns]")
```

Comment:
Let's keep the comments that was originally on top of this:
```suggestion
        # Convert ping time to 1D numpy array, stored in dict indexed by channel,
        #  this will help merge data from all channels into a cube
        for ch, val in self.ping_time.items():
            self.ping_time[ch] = np.array(val, dtype="datetime64[ns]")
```